test/common.coffee

```diff
@@ -6,6 +6,7 @@ config =
   port: 27017
   db: '_mm'
   collection: '_migrations'
+  timeout: 10
```

Comment:
The value here is quite small, I'm afraid that could make other cases fail in case of some minor glitches. What do you think?